src/components/form-elements/draft-js-mention-selector/DraftJSMentionSelector.js

```diff
@@ -421,19 +453,20 @@ class DraftJSMentionSelector extends React.Component<Props, State> {
         this.handleValidityStateUpdateHandler();
     };
 
-    getVideoTimestamp = () => {
+    getVideoTimestamp = (): { timestamp: string, timestampInMilliseconds: number } => {
         const mediaDashContainer: ?HTMLElement = document.querySelector('.bp-media-dash');
         // $FlowFixMe
         const video: ?HTMLVideoElement = mediaDashContainer?.querySelector('video');
 
+        const currentTime = video?.currentTime || 0;
+
         // $FlowFixMe
-        const totalSeconds = Math.floor(video?.currentTime || 0);
+        const totalSeconds = Math.floor(currentTime || 0);
+        const timestampToDisplay = convertSecondsToHMMSS(totalSeconds);
```

Comment:
Could we add some comments around why we need more precision with timestampInMilliseconds vs less precise timestampToDisplay

Reply:
added